mozfun/event_analysis/create_events_view/README.md

```diff
@@ -0,0 +1,181 @@
+### Usage
+
+```
```

Comment:
Adding a document type to these codeblocks will help the renderer syntax highlight them

```suggestion
```sql
```

(I'm only making the suggestion here but the same applies throughout the PR)